frontend/src/routes/_authenticated/orders/index.tsx

```diff
@@ -15,12 +16,39 @@ export const Route = createFileRoute("/_authenticated/orders/")({
 });
 
 export function Orders() {
-  const orders: Order[] = Route.useLoaderData();
+  const fullOrders = Route.useLoaderData()!;
+  const [query, setQuery] = useState<string>("");
+
+  const orderFilterPredicate = (
+    order: OrderPublicWithProviders,
+    queryBy: string,
+  ) => {
+    // TODO: We'd probably like more sophisticated search later, but it's good enough for now
+    const flattenedOrder =
```

Comment:
not sure, business decision:
a single user can be a manager for multiple providers right? So maybe search by provider also makes sense. But probably that would be best achieved via filter